www/api/index

```diff
@@ -87,9 +109,16 @@ tags associated with an IFDB listing.
 <p><a href="taggame"><b>taggame</b></a>: Edit tags on an IFDB
 listing.
 
+<h2>JSON is preferred for most APIs</h2>
+
+<p>Most of our public APIs have the option to use XML or JSON, but we recommend that
```

Comment:
Mention that the XML API is deprecated, and in some cases very different than the JSON one...?

I'm particularly thinking of taggame.

Reply:
I guess the XML API for `search` is "deprecated," but IMO "deprecated" means "warning, avoid, don't use" and I don't think that applies to `viewgame?ifiction`. That's going to have legitimate uses for as long as the Treaty of Babel is a thing.

So, for now, I'm still preferring to say "JSON is preferred" rather than "XML is deprecated."

Now, for tags, that's a separate PR #1077, and I think we've got some thinking to do around that, but my approach there is to _only_ document the JSON version of `taggame`.